src/staticfile/integration/integration_suite_test.go

```diff
@@ -17,27 +19,48 @@ import (
 
 var bpDir string
 var buildpackVersion string
+var buildpackZipFile string
 var packagedBuildpack cutlass.VersionedBuildpackPackage
 
 func init() {
-	flag.StringVar(&buildpackVersion, "version", "", "version to use (builds if empty)")
+	flag.StringVar(&buildpackVersion, "version", "", "version to use (builds if version and zipFile empty)")
 	flag.BoolVar(&cutlass.Cached, "cached", true, "cached buildpack")
 	flag.StringVar(&cutlass.DefaultMemory, "memory", "128M", "default memory for pushed apps")
 	flag.StringVar(&cutlass.DefaultDisk, "disk", "128M", "default disk for pushed apps")
+	flag.StringVar(&buildpackZipFile, "zipFile", "", "buildpack zip file in buildpack root dir (builds if zipFile and version empty)")
 	flag.Parse()
 	fmt.Println("cutlass.Cached", cutlass.Cached)
 }
 
 var _ = SynchronizedBeforeSuite(func() []byte {
 	// Run once
-	if buildpackVersion == "" {
+	if buildpackVersion == "" && buildpackZipFile == "" {
 		packagedBuildpack, err := cutlass.PackageUniquelyVersionedBuildpack()
 		Expect(err).NotTo(HaveOccurred())
 
 		data, err := json.Marshal(packagedBuildpack)
 		Expect(err).NotTo(HaveOccurred())
 		return data
 	}
+	if buildpackZipFile != "" {
+		Expect(buildpackVersion).NotTo(BeEmpty())
+
+		rootDir, err := cutlass.FindRoot()
+		Expect(err).NotTo(HaveOccurred())
+
+		buildpackZipFilePath := filepath.Join(rootDir, buildpackZipFile)
```

Comment:
Tools that require files to be in specific folders can be pretty frustrating. What would you think of allowing absolute paths by only doing this `filepath.Join` if the `buildpackZipFile` is not already absolute? (i.e. with `filepath.IsAbs`).